tests/generative/variable_strategies.py

```diff
@@ -128,51 +128,35 @@ def count(_type, _frame):
     @st.composite
     def is_null(draw, _type, frame):
         type_ = draw(any_type())
-        return Function.IsNull(
-            draw(series(type_, draw(one_row_per_patient_frame_or(frame))))
-        )
+        return Function.IsNull(draw(series(type_, frame)))
 
     @st.composite
     def not_(draw, type_, frame):
-        return Function.Not(
-            draw(series(type_, draw(one_row_per_patient_frame_or(frame))))
-        )
+        return Function.Not(draw(series(type_, frame)))
```

Comment:
When you've only got one `draw()` left (here and elsewhere) you don't need a `@st.composite`.

```
def not_(type_, frame):
    return st.builds(Function.Not, series(type_, frame))
```